### PR TITLE
Update deprecated tests syntax

### PR DESCRIPTION
**Pull request by ChiCoTran**
Refs #26132

### CODE REVIEW

**Comment by lindsayad** [comment]
@cpgr if you want some help looking over the MOOSE test directory, let me know. I'm going to unsubscribe for now. Mention me if you want help

**Review by MengnanLi91 — COMMENTED** [review]
Thanks for cleaning up all these old tests! Overall looks good, just a few minor things

File: test/tests/transfers/multiapp_nearest_node_transfer/cached_multiple_apps/main.i [hunk 1]
```diff
@@ -95,29 +95,25 @@
     to_multi_app = TF_sub
     source_variable = Layered_Average
     variable = Temperature
-    fixed_meshes = True
   []
   [to_sub_layers_elem]
     type = MultiAppNearestNodeTransfer
```
[the remark this answers is not in the document]
Comment (ChiCoTran) [reply test/tests/transfers/multiapp_nearest_node_transfer/cached_multiple_apps/main.i:100]: Causing deprecated error.

**Review by MengnanLi91 — APPROVED** [review]
Looks all good to me. Thanks for the fix!

**Pull request merged**